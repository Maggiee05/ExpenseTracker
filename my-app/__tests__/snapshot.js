import React from 'react';
import renderer from 'react-test-renderer';

import LoginScreen from '../screens/login_screen';
import RegisterScreen from '../screens/register_screen';

describe('App', () => {
  it('renders login screen', async () => {
    const tree = renderer.create(<LoginScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders register screen', async () => {
    const tree = renderer.create(<RegisterScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
