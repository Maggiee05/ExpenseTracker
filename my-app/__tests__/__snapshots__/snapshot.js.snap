// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders login screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 60,
        "fontWeight": "700",
        "marginBottom": "10%",
      }
    }
  >
    Hello
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 8,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": "5%",
        "width": 250,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Username"
      rejectResponderTermination={true}
      style={
        Object {
          "height": 40,
          "width": 200,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 8,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": "5%",
        "width": 250,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Password"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "height": 40,
          "width": 200,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#6495ed",
        "borderRadius": 8,
        "height": 35,
        "marginBottom": "3%",
        "marginTop": "5%",
        "opacity": 1,
        "width": 150,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#FFFFFF",
          "fontSize": 16,
          "fontWeight": "bold",
          "justifyContent": "center",
          "marginTop": "5%",
        }
      }
    >
      LOGIN
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#6495ed",
        "borderRadius": 8,
        "height": 35,
        "marginBottom": "3%",
        "opacity": 1,
        "width": 150,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#FFFFFF",
          "fontSize": 16,
          "fontWeight": "bold",
          "justifyContent": "center",
          "marginTop": "5%",
        }
      }
    >
      REGISTER
    </Text>
  </View>
</View>
`;

exports[`App renders register screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 8,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": "5%",
        "width": 250,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Username"
      rejectResponderTermination={true}
      style={
        Object {
          "height": 40,
          "width": 200,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 8,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": "5%",
        "width": 250,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Password"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "height": 40,
          "width": 200,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderRadius": 8,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "center",
        "marginBottom": "5%",
        "width": 250,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      onChangeText={[Function]}
      placeholder="Confirm Password"
      rejectResponderTermination={true}
      secureTextEntry={true}
      style={
        Object {
          "height": 40,
          "width": 200,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#66cdaa",
        "borderRadius": 8,
        "height": 35,
        "marginBottom": "3%",
        "marginTop": "5%",
        "opacity": 1,
        "width": 150,
      }
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "#FFFFFF",
          "fontSize": 16,
          "fontWeight": "bold",
          "justifyContent": "center",
          "marginTop": "5%",
        }
      }
    >
      SIGN UP
    </Text>
  </View>
</View>
`;
